src/boot_O2_g3/syncprintf.c

```diff
@@ -1,10 +1,11 @@
 #include "global.h"
 
-// This file contains print functions that are used by fault.c, so they are placed in the fault module
-
-void __osSyncVPrintf(UNK_TYPE4 param_1, ...) {
+// This isn't a vprintf since it doesn't take a va_list
+void __osSyncVPrintf(const char* fmt, ...) {
 }
-void osSyncPrintf(char* str, ...) {
+
+void osSyncPrintf(const char* fmt, ...) {
 }
-void rmonPrintf(UNK_TYPE4 param_1, ...) {
+
+void rmonPrintf(const char* fmt, ...) {
 }
```

Comment:
Two things about this file:
1. I assume this is that IDO stubbing we were discussing earlier?
2. Is the first function incorrectly-named, then?

Reply:
I think this is manual stubbing, for debug vs non-debug.
The first function appears to be incorrectly named, since `const char* fmt, va_list args` doesn't match. But looking at the debug sources, that "should" be what that function is, so it's weird. It's not called anywhere either, so we can't fall back to that for a good name either.

Reply:
I suppose it doesn't really matter if it's not actually used, then. I think retail OoT has a similar file in boot.